### PR TITLE
Create Wikipedia cookbook

### PR DESCRIPTION
**Pull request by justinqcai**
This PR creates the `wikipedia_assistant.py` file in the `cookbook/knowledge` directory to serve as an example for users using the WikipediaKnowledgeBase.

Issue: https://github.com/phidatahq/phidata/issues/91

### CODE REVIEW

File: cookbook/knowledge/wikipedia_assistant.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from phi.knowledge.wikipedia import WikipediaKnowledgeBase
+from phi.vectordb.pgvector import PgVector2
+from phi.assistant import Assistant
+
+from resources import vector_db
+
+# install `wikipedia` via `pip install wikipedia` first
```
Comment (ysolanky) [inline cookbook/knowledge/wikipedia_assistant.py:7]: This comment should be removed

**Comment by justinqcai** [comment]
@ysolanky Thanks! And the reason I don't have it as Wikipedia.py is because the name of the original wikipedia class is also wikipedia.py - when the WikipediaKnowledgeBase tries to import 'wikipedia', it runs into a circular dependency error and fails to run.

**Comment by ysolanky** [comment]
You're right @justinqcai we cannot name this file wikipedia.py :)

**Review by ashpreetbedi — APPROVED** [review]

**Comment by ashpreetbedi** [comment]
Thank you for the contribution @justinqcai !

**Pull request merged**